.buildkite/pipeline.quick.yml

```diff
@@ -134,9 +131,6 @@ steps:
           - "--app=/app/build/release/fixture-r21.apk"
           - "--farm=bs"
           - "--device=ANDROID_9_0"
-          - "--retry=1"
-          - "--strict-undefined"
-          - "--strict-pending"
```

Comment:
What is the effect of removing these flags?

Reply:
It means that failing scenarios won't retry.  I don't think these should have been in the Android 9 run at all in the first instance and appear to have been a copy/paste or perhaps merge error.

Reply:
Are the `--strict-` flags part of that or something else?

Reply:
Yeah, they are part of it.  There are three flavours of "strict" - undefined, pending and flaky.  MazeRunner makes them all strict by default, but flaky is not compatible with retry (if you want the retry to pass the test).... so the two strict flags were added with retry to make it as strict as it could be.